src/coreComponents/physicsSolvers/contact/SolidMechanicsLagrangeContact.cpp

```diff
@@ -695,43 +699,77 @@ real64 SolidMechanicsLagrangeContact::calculateContactResidualNorm( DomainPartit
     {
       arrayView1d< globalIndex const > const & dofNumber = subRegion.getReference< array1d< globalIndex > >( dofKey );
       arrayView1d< integer const > const & ghostRank = subRegion.ghostRank();
+      arrayView1d< integer const > const & fractureState = subRegion.getField< contact::fractureState >();
+      arrayView1d< real64 const > const & area = subRegion.getElementArea();
 
-      RAJA::ReduceSum< parallelHostReduce, real64 > localSum( 0.0 );
+      RAJA::ReduceSum< parallelHostReduce, real64 > stickSum( 0.0 );
+      RAJA::ReduceSum< parallelHostReduce, real64 > slipSum( 0.0 );
+      RAJA::ReduceMax< parallelHostReduce, real64 > slipMax( 0.0 );
+      RAJA::ReduceSum< parallelHostReduce, real64 > openSum( 0.0 );
+      RAJA::ReduceMax< parallelHostReduce, real64 > openMax( 0.0 );
       forAll< parallelHostPolicy >( subRegion.size(), [=] ( localIndex const k )
       {
         if( ghostRank[k] < 0 )
         {
           localIndex const localRow = LvArray::integerConversion< localIndex >( dofNumber[k] - rankOffset );
-          for( localIndex dim = 0; dim < 3; ++dim )
+          switch( fractureState[k] )
           {
-            localSum += localRhs[localRow + dim] * localRhs[localRow + dim];
+            case contact::FractureState::Stick:
+              {
+                for( localIndex dim = 0; dim < 3; ++dim )
+                {
+                  real64 const norm = localRhs[localRow + dim] / area[k];
```

Comment:
why dividing by an area? the residual has the units of a cubic length so I would expect the normalizer to also have those units so that we keep everything dimensionless. We could use the minimum aperture or something like that. It's not that obvious for the tangential component though. Maybe the max jump at the previous timestep?

Reply:
I didn't find any good "reference aperture" value which is not too small to normalize by
So after discussion with @Guotong-Ren we decided that 1m as reference is not that bad